app/models/artsy_tags_updater.rb

```diff
@@ -1,12 +1,12 @@
 class ArtsyTagsUpdater
-  def self.update(artwork_id, tags_to_add = [], tags_to_remove = [])
-    new(artwork_id, tags_to_add, tags_to_remove).update
+  def self.update(artwork_id, tags_whitelist = [], tags_blacklist = [])
+    new(artwork_id, tags_whitelist, tags_blacklist).update
   end
 
-  def initialize(artwork_id, tags_to_add, tags_to_remove)
+  def initialize(artwork_id, tags_whitelist, tags_blacklist)
     @artwork_id = artwork_id
-    @tags_to_add = tags_to_add
-    @tags_to_remove = tags_to_remove
+    @tags_whitelist = tags_whitelist
+    @tags_blacklist = tags_blacklist
```

Comment:
As configured, this would result in the following error from Rubocop:

<img width="1439" alt="Screen Shot 2021-12-01 at 6 22 45 PM" src="https://user-images.githubusercontent.com/140521/144687114-96425333-66c8-4593-8756-d14c54dd55c3.png">

Reply:
@anandaroop I just remembered this might not be sufficient. What is also happening is that our dependencies sometimes have not been updated. Is there concise language we could add here? From the RFC on inclusive language I suggest, and engineering directors/EVP agreed, that we should feel comfortable either addressing the issues ourselves on the dependency, opening pull requests, or submitting issues, depending on the size/culture of the third party system. If it's a dependency we are using that has the non-inclusive language, can we suppress the lint warning, do we live with it? etc.

Reply:
In special scenarios (when there is a reason that makes that specific part of the code too hard to be changed), I would say we handle this as we would handle any other issue pointed out by the linter. We could disable the rule only at that point so it won't block our work (in parallel, we can open PRs for the dependency, submit issues, etc).

Reply:
But I think we should update the linter language here to point out updating a dependency.

Reply:
I know - this was the problem discussed at the management level. It's not so much us as our dependencies, and the time needed to address them. However, IMO leaving very outdated dependencies is its own business risk. The harder it gets to update, the more serious we should take updating. (Tragedy of the commons.)